lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -190,8 +190,8 @@ fn do_test_monitor_temporary_update_fail(disconnect_count: usize) {
 					_ => panic!("Unexpected event"),
 				}
 
-				if let Err(msgs::LightningError{err, action: msgs::ErrorAction::IgnoreError }) = nodes[0].node.handle_commitment_signed(&nodes[1].node.get_our_node_id(), commitment_signed) {
-					assert_eq!(err, "Previous monitor update failure prevented generation of RAA");
+				nodes[0].node.handle_commitment_signed(&nodes[1].node.get_our_node_id(), commitment_signed);
+				if let MessageSendEvent::HandleError { .. } = nodes[0].node.get_and_clear_pending_msg_events()[0] {
```

Comment:
Can you restore the error-message-checking that was here previously?

Reply:
Looks like this is still missing the IgnoreError check that was here? Also, why not check the error message inside the HandleError?